v0/destinations/fb_custom_audience/transform.js

```diff
@@ -111,27 +114,30 @@ const ensureApplicableFormat = (userProperty, userInformation) => {
     case "FI":
       if (userProperty !== "FI") {
         updatedProperty = userInformation
+          .toString()
           .toLowerCase()
           .replace(/[^a-zA-Z@#$%&!]/g, "");
       } else {
         updatedProperty = userInformation
+          .toString()
           .toLowerCase()
           .replace(/[^a-zA-Z@#$%&!,.?]/g, "");
       }
       break;
     case "MADID":
-      updatedProperty = userInformation.toLowerCase();
+      updatedProperty = userInformation.toString().toLowerCase();
       break;
     case "COUNTRY":
-      updatedProperty = userInformation.toLowerCase();
+      updatedProperty = userInformation.toString().toLowerCase();
       break;
     case "ZIP":
-      userInformationTrimmed = userInformation.replace(/\s/g, "");
+      userInformationTrimmed = userInformation.toString().replace(/\s/g, "");
       updatedProperty = userInformationTrimmed.toLowerCase();
       break;
     case "ST":
     case "CT":
       updatedProperty = userInformation
```

Comment:
Apart from calling toString so many times we can directly make `userInformation` a string where we are calling `ensureApplicableFormat` function.

Reply:
yeah we can 😅